colabs/wandb_registry/zoo_wandb.ipynb

```diff
@@ -0,0 +1,634 @@
+{
```

Comment:
<p>Since we are logging tensors, there is an opportunity to call this collection "zoo-dataset-tensors" or similar, within the core Datasets registry, which would differ, for example, from "zoo-dataset-tensors-split" created in the next section</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/examples/pull/560/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>